core-api/src/main/resources/definitions/exposed-services.json

```diff
@@ -296,15 +296,15 @@
     },
     {
       "name": "NIFI_REGISTRY",
-      "displayName": "Nifi Registry",
+      "displayName": "NiFi Registry",
       "serviceName": "NIFI_REGISTRY_SERVER",
-      "knoxService": "NIFI-REGISTRY",
```

Comment:
I don't think this change is right? It should be `NIFI-REGISTRY` and not with an underscore?

References:
* https://github.com/apache/knox/blob/master/gateway-service-definitions/src/main/resources/services/nifi-registry/0.5.0/service.xml#L17
* https://github.com/hortonworks/cloudbreak/blob/master/orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2#L624